### PR TITLE
Correct permissions for index.html and robots.txt on new added web domain

### PR DESCRIPTION
**Pull request by moucho**
The right permissions for website files are 644, not 755 as are being set now

### CODE REVIEW

File: bin/v-add-web-domain [hunk 1]
```diff
@@ -90,6 +90,7 @@ chown root:$user /var/log/$WEB_SYSTEM/domains/$domain.* $conf
 chmod 640 /var/log/$WEB_SYSTEM/domains/$domain.*
 chmod 751 $HOMEDIR/$user/web/$domain $HOMEDIR/$user/web/$domain/*
 chmod 551 $HOMEDIR/$user/web/$domain/stats $HOMEDIR/$user/web/$domain/logs
+chmod 644 $HOMEDIR/$user/web/$domain/public_html/*
```
Comment (joubertredrat) [inline bin/v-add-web-domain]: I think that is good set for `public_shtml` too

**Comment by anton-reutov** [comment]
What about public_shtml ?

**Comment by anton-reutov** [comment]
What about domains after installation?

**Comment by anton-reutov** [comment]
Please create new PR with this changes

**Pull request merged**